client/client.go

```diff
@@ -187,6 +191,8 @@ func NewClient(cfg *config.Config, consulCatalog consul.CatalogAPI, consulServic
 		serversDiscoveredCh: make(chan struct{}),
 	}
 
+	c.baseLabels = []metrics.Label{{"node_id", c.Node().ID}, {"datacenter", c.Node().Datacenter}}
```

Comment:
would it be possible to include region too? would make it easier to work with the metrics inside datadog and influx when you have one common metrics storage cross env

Reply:
Including the region will take a bit more work and is outside the scope for this PR, but we will definitely keep this in mind for the future. Thanks for the suggestion!